bitcoin_tools/analysis/status/utils.py

```diff
@@ -377,6 +385,48 @@ def decode_utxo_v08_v014(utxo):
 
     return {'version': version, 'coinbase': coinbase, 'outs': outs, 'height': height}
 
+def decompress_script(script_type,script_bytes):
```

Comment:
I like the idea of this function, however doing it from scratch is not necessary. You can use some of the tools of the library to avoid adding new dependencies. Check [OutputScript](https://github.com/sr-gi/bitcoin_tools/blob/master/bitcoin_tools/core/script.py) class and all its constructors to create the scripts.